pulp_rpm/app/migrations/0040_disttree_digest.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.9 on 2021-12-20 22:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rpm', '0039_rpmalternatecontentsource'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='distributiontree',
+            name='digest',
+            field=models.CharField(default='should be removed from a repo version with a sync', max_length=64),
```

Comment:
There is no way for us to fix the digest properly, since there might be a wrong artifact associated with the content.
The idea is that with the subsequest sync (either optimize=False or any sync if there are upstream changes), new content units will be created and the ones with this weird "digest" will become orphaned and be cleaned up at some point.
Any clearer sentence twhich fits 64chars to put into the field for migration?

Reply:
That's unfortunate but makes perfect sense.  What about additive sync?  It still gets purged in that case, yeah?

>Any clearer sentence which fits 64chars to put into the field for migration?

The digest of the file makes the most sense to me.  I don't think they change that often so I doubt it would churn too much.

Reply:
> That's unfortunate but makes perfect sense. What about additive sync? It still gets purged in that case, yeah?
Yes, because only one distribution tree is allowed in repo version.

> 
> > Any clearer sentence which fits 64chars to put into the field for migration?
> 
> The digest of the file makes the most sense to me. I don't think they change that often so I doubt it would churn too much.

I can't put the digest of the file, it might be a wrong file associated with the content, so this content unit will never be replaced then. So I'm putting some wording there that this content should eventually go away.